components/ActivityIndicator/examples/ActivityIndicatorExample.swift

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Nit: Please add newlines to the copyright section to match AnimationTimingExample.h/.m.  It makes it a little easier to read.